src/App.jsx

```diff
@@ -1,16 +1,36 @@
-import React from 'react';
-import { registerRootComponent } from 'expo';
+import React, { useState, useEffect } from 'react';
+import { registerRootComponent, AppLoading } from 'expo';
+import * as Font from 'expo-font';
 import { Provider } from 'react-redux';
 import { createAppContainer } from 'react-navigation';
 import AppNavigator from './Navigator';
-import store from './redux/store';
+import store from './reduxStore';
+import Roboto from '../assets/Fonts/Roboto/Roboto-Regular.ttf';
+import RobotoMedium from '../assets/Fonts/Roboto/Roboto-Medium.ttf';
 
 const AppContainer = createAppContainer(AppNavigator);
 
-const App = () => (
-  <Provider store={store}>
-    <AppContainer />
-  </Provider>
-);
+const App = () => {
+  const [fontLoading, setFontLoading] = useState(true);
+  useEffect(() => {
+    async function loadFont() {
+      await Font.loadAsync({
```

Comment:
Cannot it throw an error? Don't we need to handle that? pls double check in the docs or somewhere

Reply:
fixed